backend/configs/usr/lib/cgi-bin/factory_reset.py

```
@@ -0,0 +1,51 @@
+    sys.stdout.write("Status: 500 Internal Server Error\r\n\r\nInternal Server Error: %s" % msg)
+    sys.exit(1)
+
+
+def check_for_form(form):
+    try:
+        form.keys()
+    except Exception:
+        _error("No form in request")
+
+
+def check_for_confirmation(form):
```

Ruff (0.8.2)

Missing return type annotation for public function `check_for_confirmation`

Add return type annotation: `None`

(ANN201)